requirements.txt

```diff
@@ -1,3 +1,2 @@
 numpy>=1.18
 matplotlib>=3.1.2
```

Comment:
I prefer to keep this requirement

Reply:
Agreed, the library requires h5py for correct functioning. We want to be able to use it without h5py for a specific use-case, and that means installing while ignoring the dependency list.

I would also reword the commit, to "Allow h5py import to fail", and explain in the commit message the expected use case.